### PR TITLE
Update SonarLint files from server

### PR DESCRIPTION
**Pull request by martin-strecker-sonarsource**

### CODE REVIEW

File: analyzers/.sonarlint/sonaranalyzer-dotnet/CSharp/SonarLint.xml [hunk 1]
```diff
@@ -171,7 +171,7 @@
       <Key>S3776</Key>
       <Parameters>
         <Parameter>
-          <Key>propertyThreshold</Key>
+          <Key>propertyThreshold </Key>
```
Comment (pavel-mikula-sonarsource) [inline analyzers/.sonarlint/sonaranalyzer-dotnet/CSharp/SonarLint.xml:174]: This should not be here, I've noticed that when playing with the QP.
Fix is on the way #5897
Please revert this change in all 3 PRs

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
We have a regression in 8.42 that's messing with this

File: analyzers/.sonarlint/sonaranalyzer-dotnet/CSharp/SonarLint.xml (hunk 1 above)
Comment (martin-strecker-sonarsource) [inline analyzers/.sonarlint/sonaranalyzer-dotnet/CSharp/SonarLint.xml:174]:
```suggestion
          <Key>propertyThreshold</Key>
```

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM
